cluster-configs/upp-delivery-content-test-eu.properties

```diff
@@ -0,0 +1,53 @@
+#Keeping the cluster name as small as possible
+CLUSTER_NAME=UppK8sDeliveryTestEu
+REGION=eu-west-1
+API_DNS_NAME=upp-k8s-delivery-test-eu-api.ft.com
+
+KMS_KEY_ARN=arn:aws:kms:eu-west-1:070529446553:key/870f9ef6-96c1-409b-874f-b4e83f7232e6
+
+API_PRIVATE_ACCESS_SG=sg-53531d2b
+DEBUG_KEY_NAME=upp-k8s-provisioning-debug
+
+CONTROLLER_INSTANCE_TYPE=t2.medium
+CONTROLLER_COUNT=1
+CONTROLLER_IAM_ROLE=arn:aws:iam::070529446553:instance-profile/upp-k8s-iam-instance-profile-roles-eu-west-1-controller-role
```

Comment:
Can we add double quotes to keep it consistent with the worker and the etcd iam role please

Reply:
Yes indeed.